core/src/main/java/org/codice/imaging/nitf/core/image/TargetId.java

```diff
@@ -123,10 +123,30 @@ public final String getCountryCode() {
     }
 
     /**
-     * {@inheritDoc}
+     * Get the text equivalent of this target identifier.
+     *
+     * @return a space-padded string containing the target identifier.
+     * @throws NitfFormatException if any of the fields are null, or are too long to fit.
      */
-    @Override
-    public final String toString() {
-        return beNumber + oSuffix + countryCode;
+    public final String getAsText() throws NitfFormatException {
```

Comment:
I think the method name 'getAsText' sounds like you're trying to retrieve an attribute called 'asText'.  I think something like 'textValue()' would be a better name.